Firmware/src/modules/tools/temperaturecontrol/TemperatureControl.h

```diff
@@ -66,6 +66,7 @@ class TemperatureControl : public Module
     SigmaDeltaPwm *heater_pin{nullptr};
 
     std::string designator;
+    std::string sensor_type;
```

Comment:
I'd prefer this to be an ENUM, that is far more memory efficient and saves comparing strings all the time.